### PR TITLE
provide executor via porting package

### PR DESCRIPTION
**Pull request by benjamin-confino**
This creates a new module called `porting` that makes it possible to reflexively import classes provided by the integrator to run the TCK. To import a class you: 

- Put a file META-INF/microprofile-telemetry-tck.properties on the classpath to point to the class.
- Put the class on the classpath when the test is running. 

Currently this is used to import an impl of `Executor` so integrators than run tests that use concurrency

### CODE REVIEW

**Comment by benjamin-confino** [comment]
fixes #140

**Review by pdudits — APPROVED** [review]
While this works a simpler implementation would have been replacing the `@Resource` with `@Inject` and require implementor to provide appropriate producer by means of e. g. Arquillian extension.

File: tck/porting/pom.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>org.eclipse.microprofile.telemetry</groupId>
+        <artifactId>microprofile-telemetry-tck-parent</artifactId>
+        <version>2.0-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>microprofile-telemetry-tck-porting</artifactId>
+    <name>MicroProfile Telemetry Metrics Porting</name>
```
Comment (Emily-Jiang) [inline tck/porting/pom.xml:26]:
```suggestion
    <name>MicroProfile Telemetry Porting Executor</name>
```

**Review by Emily-Jiang — CHANGES_REQUESTED** [review]
You need to copy all of the files under resources/META-INF including the empty file `tck`

File: tck/tracing/pom.xml [hunk 2]
```diff
@@ -26,6 +26,11 @@
     <name>MicroProfile Telemetry Tracing TCK</name>
 
     <dependencies>
+        <dependency>
```
Comment (Emily-Jiang) [inline tck/tracing/pom.xml:29]: These lines should be deleted.

File: tck/tracing/README.adoc [hunk 3]
```diff
@@ -58,7 +58,7 @@ These tests test the B3 and Jaeger propagation formats which are not required. I
 
 * `optional-jaxrs-tests` JAX-RS server async programming models
 +
-Although support for JAX-RS server async programming models is not optional, these tests depend on Jakarta Concurrency because they use `ManagedExecutorService`.
+Although support for JAX-RS server async programming models is not optional, these tests require you to provide an exector. <<Executor, See the section on providing an executor>>
```
Comment (donbourne) [inline tck/tracing/README.adoc:61]: exector -> executor

**Pull request merged**